content/ko/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -0,0 +1,761 @@
+
+프로젝트 쿼터를 사용하려면, 다음을 수행해야 한다.
+
+* kubelet 구성에서 `LocalStorageCapacityIsolationFSQuotaMonitoring=true`
+  [기능 게이트](/docs/reference/command-line-tools-reference/feature-gates/)를
+  활성화한다.
+
+* 루트 파일시스템(또는 선택적인 런타임 파일시스템)에
+  프로젝트 쿼터가 활성화되어 있는지 확인한다. 모든 XFS 파일시스템은 프로젝트 쿼터를 지원한다.
+  ext4 파일시스템의 경우, 파일시스템이 마운트되지 않은 상태에서 프로젝트 쿼터
+  추적 기능을 활성화해야 한다.
+  ```bash
+  # ext4인 /dev/block-device가 마운트되지 않은 경우
+  sudo tune2fs -O project -Q prjquota /dev/block-device
+  ```
+
+* 루트 파일시스템(또는 선택적인 런타임 파일시스템)은 프로젝트 쿼터를
+  활성화한 상태에서 마운트해야 힌다. XFS와 ext4fs 모두에서,
+  마운트 옵션의 이름은 `prjquota` 이다.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## 확장된 리소스
+
+확장된 리소스는 `kubernetes.io` 도메인 외부의 전체 주소(fully-qualified)
+리소스 이름이다. 쿠버네티스에 내장되지 않은 리소스를 클러스터 운영자가 알리고
+사용자는 사용할 수 있다.
+
+확장된 리소스를 사용하려면 두 단계가 필요한다. 먼저, 클러스터
+운영자는 확장된 리소스를 알려야 한다. 둘째, 사용자는 파드의
+확장된 리소스를 요청해야 한다.
+
+### 확장된 리소스 관리
+
+#### 노드-레벨의 확장된 리소스
+
+노드-레벨의 확장된 리소스는 노드에 연결된다.
+
+##### 장치 플러그인 관리 리소스
+각 노드에서
+장치 플러그인 관리 리소스를 알리는 방법은
+[장치 플러그인](/ko/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/)을 참조한다.
+
+##### 기타 리소스
+새로운 노드-레벨의 확장된 리소스를 알리기 위해, 클러스터 운영자는
+API 서버에 `PATCH` HTTP 요청을 제출하여 클러스터의
+노드에 대해 `status.capacity` 에서 사용할 수 있는 수량을 지정할 수 있다. 이 작업
+후에는, 노드의 `status.capacity` 에 새로운 리소스가 포함된다. 이
+`status.allocatable` 필드는 kubelet에 의해 비동기적으로 새로운
+리소스로 자동 업데이트된다. 참고로 스케줄러가 파드 적합성을 평가할 때 노드
+`status.allocatable` 값을 사용하므로, 노드 용량을
+새 리소스로 패치하는 것과 해당 노드에서 리소스를 스케줄하도록 요청하는 첫 번째 파드
+사이에 약간의 지연이 있을 수 있다.
+
+**예제:**
+
+다음은 `curl` 을 사용하여 마스터가 `k8s-master` 인 노드 `k8s-node-1` 에
+5개의 "example.com/foo" 리소스를 알리는 HTTP 요청을 구성하는 방법을
+보여주는 예이다.
+
+```shell
+curl --header "Content-Type: application/json-patch+json" \
+--request PATCH \
+--data '[{"op": "add", "path": "/status/capacity/example.com~1foo", "value": "5"}]' \
+http://k8s-master:8080/api/v1/nodes/k8s-node-1/status
+```
+
+{{< note >}}
+앞의 요청에서, `~1` 은 패치 경로에서 문자 `/` 의
+인코딩이다. JSON-Patch의 작업 경로 값은
+JSON-Pointer로 해석된다. 더 자세한 내용은,
+[IETF RFC 6901, 섹션 3](https://tools.ietf.org/html/rfc6901#section-3)을 참조한다.
+{{< /note >}}
+
+#### 클러스터-레벨의 확장된 리소스
+
+클러스터-레벨의 확장된 리소스는 노드에 연결되지 않는다. 이들은 일반적으로
+리소스 소비와 리소스 쿼터를 처리하는 스케줄러 익스텐더(extender)에 의해 관리된다.
+
+[스케줄러 정책 구성](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/scheduler/api/v1/types.go#L31)에서
+스케줄러 익스텐더가
+처리하는 확장된 리소스를 지정할 수 있다.
+
+**예제:**
+
+스케줄러 정책에 대한 다음의 구성은 클러스터-레벨의 확장된 리소스
+"example.com/foo"가 스케줄러 익스텐더에 의해 처리됨을
+나타낸다.
+
+- 파드가 "example.com/foo"를 요청하는 경우에만 스케줄러가 파드를 스케줄러
+     익스텐더로 보낸다.
+- 이 `ignoredByScheduler` 필드는 스케줄러가 `PodFitsResources` 속성에서
+     "example.com/foo" 리소스를 확인하지 않도록 지정한다.
+
+```json
+{
+  "kind": "Policy",
+  "apiVersion": "v1",
+  "extenders": [
+    {
+      "urlPrefix":"<extender-endpoint>",
+      "bindVerb": "bind",
+      "managedResources": [
+        {
+          "name": "example.com/foo",
+          "ignoredByScheduler": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+### 확장된 리소스 소비
+
+사용자는 CPU와 메모리 같은 파드 스펙의 확장된 리소스를 사용할 수 있다.
+스케줄러는 리소스 계정을 관리하여 사용 가능한 양보다
+많은 양이 동시에 파드에 할당되지 않도록 한다.
```

Comment:
```suggestion
많은 양을 파드에 동시에 할당되지 않도록 한다.
```

Reply:
> The scheduler takes care of the resource accounting so that no more than the
> available amount is simultaneously allocated to Pods.

```suggestion
스케줄러는 리소스 어카운팅(resource accounting)을 처리하여 사용 가능한 양보다
많은 양의 리소스가 여러 파드에 동시에 할당되지 않도록 한다.
```

부서분들 세 명이 모여 고민했습니다. :-)
`리소스 계정` 이라는 표현이 어색하여 `리소스 어카운팅` 이라 쓰고 한영병기하는 것이 나을 듯 합니다. `많은 양` 보다는 `리소스` 를 포함하여 쓰는 것이 이해하기가 더 좋을 것 같습니다.

이 의견은 어떤까요?

Reply:
네, 병행 표기가 좋아 보입니다만,  `리소스 계정을 관리하여`, `리소스 어카운팅(resource accounting)을 처리하여`보다 `리소스 수량을 관리하여` 는 어떠신지?

Reply:
`resource accounting` 의 의미가 리소스 사용을 기록하고 관리한다는 의미로 받아들여져서 `처리` 한다고 했는데 말씀하신 것처럼 `관리하여` 로 표현해도 괜찮을 것 같습니다.